src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -425,6 +425,17 @@ def get_logs(self) -> WorkflowLogs:
         """
         return self._runtime.get_workflow_logs(wf_run_id=self.run_id)
 
+    def get_system_logs(self) -> t.List[str]:
+        """
+        Unstable: this API will change.
+
+        Returns all available system logs printed during execution of this workflow run.
+
+        Returns:
+            A list of log lines printed when running this workflow.
+        """
+        return self._runtime.get_system_logs(wf_run_id=self.run_id)
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Add system logs to the `WorkflowLogs` object returned by `get_logs`